tests/auth-bypassed/phase1-core/grafana-version.spec.ts

```diff
@@ -1,9 +1,10 @@
 import { test, expect } from '@playwright/test';
 
 test('Check Grafana Version from Help Button', async ({ page }) => {
-  await page.goto('http://localhost:3000/', {waitUntil: 'networkidle'});
-  // only works from v10+
-  await page.getByLabel('Help').click();
+  await page.goto('http://localhost:3000/');
+  const locator = page.getByRole('button', { name: 'Help' });
+  await locator.waitFor();
```

Comment:
[nitpick]: waitFor doesn't seem to be recommended by [Playwright](https://playwright.dev/docs/testing-library#replacing-waitfor) instead we should use assertions that have auto waits: https://playwright.dev/docs/test-assertions#auto-retrying-assertions